frontend/src/api/project.js

```diff
@@ -12,6 +12,7 @@ const getProject = (projectId) => {
     return client.get(`/api/v1/projects/${projectId}`).then(res => {
         res.data.createdSince = daysSince(res.data.createdAt)
         res.data.updatedSince = daysSince(res.data.updatedAt)
+        res.data.flowLastUpdatedSince = daysSince(res.data.flowLastUpdatedAt)
```

Comment:
Does this handle a null/undefined value?

Reply:
It just returns empty string https://github.com/flowforge/flowforge/blob/619565bcb9dcb9213cd217d12502fd0164c47a88/frontend/src/utils/daysSince.js#L3-L6